doc/Tex/RAM_SCB_testing.tex

```diff
@@ -1,77 +1,57 @@
-RAM-SCB comes packaged with a test suite to determine if the current 
-installation operates correctly and produces the expected results.  
-Different tests evaluate different code capabilities and options.  
-Testing is an especially powerful development tool for evaluating the impact, 
-intentional or not, of changes to the source code.  A summary of the tests
-and associated commands can be generated by using {\tt make test\_help}.
+RAM-SCB comes packaged with a test suite to determine if the current installation operates correctly and produces the expected results. Different tests evaluate different code capabilities and options. Testing is an especially powerful development tool for evaluating the impact, intentional or not, of changes to the source code. A summary of the tests and associated commands can be generated by using {\tt make test\_help}.
```

Comment:
We should tell the user to check current test status by visiting the github repo.